client/blocks/nps-survey/docs/example.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import NpsSurvey from '../';
+
+class NpsSurveyExample extends PureComponent {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			wasDismissed: false,
+			wasSubmitted: false,
+			surveyName: null,
+			recommendationValue: null,
+		};
+		this.handleDismissed = this.handleDismissed.bind( this );
+	}
```

Comment:
Drive-by tip: with ES6+ instance properties, this `constructor` definition could be replaced like so:

```js
class NpsSurveyExample extends PureComponent {
  // static indicates a class property (i.e., cross-instance)
  static displayName = 'NpsSurvey';

  // instance prop eval'd upon instantiation
  state = {
    wasDismissed: false,
    // …
  };

  // same, but thanks to the fat arrow we get a bound method
  handleDismissed = ( event ) => {
    this.setState( … );
  }
```